crates/re_viewer/src/app.rs

```diff
@@ -400,6 +400,14 @@ impl App {
                     );
                 }
             }
+
+            SystemCommand::SetSelection(store_id, item) => {
```

Comment:
This makes me wonder if we should move more over to this system command. If everything where to use it we wouldn't need double buffering on the selection state 🤔

Reply:
Yup, I think we should